<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P_Shop</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Res/img/Logo.png</normaloff>:/Res/img/Logo.png</iconset>
  </property>
  <widget class="QLabel" name="lbl_border">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>980</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_search">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(244,159,30);</string>
   </property>
   <property name="text">
    <string>Search :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_search">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(244,159,30);
color: rgb(47, 0, 255);</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GP_SearchMethod">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>250</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(244,159,30);
</string>
   </property>
   <property name="title">
    <string>Search by :</string>
   </property>
   <widget class="QRadioButton" name="radio_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="redio_Category">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tbl_search">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>250</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(244,159,30);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_0">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 5px solid rgb(244,159,30);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_1">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>100</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(244,159,30);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>240</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(244,159,30);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>380</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(244,159,30);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(32, 0, 140);
background-color: rgb(244,159,30);
</string>
   </property>
   <property name="text">
    <string>&amp;Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(32, 0, 140);
background-color: rgb(244,159,30);
</string>
   </property>
   <property name="text">
    <string>&amp;Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_help">
   <property name="geometry">
    <rect>
     <x>815</x>
     <y>60</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(32, 0, 140);
background-color: rgb(244,159,30);
</string>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_description">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>510</y>
     <width>661</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(244,159,30);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_BG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 0, 140);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_warning">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(244,159,30);</string>
   </property>
   <property name="text">
    <string>You are guest user

Login or Register</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(244,159,30);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>380</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(244,159,30);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_6">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 4px solid rgb(244,159,30);

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lbl_BG</zorder>
  <zorder>lbl_border</zorder>
  <zorder>lbl_search</zorder>
  <zorder>txt_search</zorder>
  <zorder>GP_SearchMethod</zorder>
  <zorder>tbl_search</zorder>
  <zorder>pic_0</zorder>
  <zorder>pic_1</zorder>
  <zorder>pic_2</zorder>
  <zorder>pic_3</zorder>
  <zorder>btn_register</zorder>
  <zorder>btn_login</zorder>
  <zorder>btn_help</zorder>
  <zorder>txt_description</zorder>
  <zorder>lbl_warning</zorder>
  <zorder>pic_4</zorder>
  <zorder>pic_5</zorder>
  <zorder>pic_6</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
